srcalc/src/test/java/gov/va/med/srcalc/vista/RpcVistaPatientDaoTest.java

```diff
@@ -157,4 +172,33 @@ public final void testLabResponseNoUnits()
         // Insure the lab was not added and no exceptions occurred.
         assertEquals(1, patient.getLabs().size());
     }
+    
+    @Test
+    public final void testHealthFactorsValid() throws Exception
+    {
+        final VistaProcedureCaller caller = mockVistaProcedureCaller();
+        when(caller.doRpc(
+                RADIOLOGIST_DUZ,
+                RemoteProcedure.GET_HEALTH_FACTORS,
+                String.valueOf(PATIENT_DFN)))
+                .thenReturn(VALID_HEALTH_FACTORS);
+        final RpcVistaPatientDao dao = new RpcVistaPatientDao(caller, RADIOLOGIST_DUZ);
+        final Patient patient = dao.getPatient(PATIENT_DFN);
+        final SimpleDateFormat format = new SimpleDateFormat("MM/dd/yy");
+        final List<HealthFactor> expectedFactors = ImmutableList.of(
+                new HealthFactor(format.parse("08/22/2014"),"DEPRESSION ASSESS POSITIVE (MDD)"),
+                new HealthFactor(format.parse("08/20/2014"), "ALCOHOL - TREATMENT REFERRAL"),
+                new HealthFactor(format.parse("07/30/2014"), "GEC HOMELESS"));
+        
+        assertEquals(expectedFactors, patient.getHealthFactors());
+    }
+    
+    @Test
+    public final void testNoHealthFactors()
+    {
+        final VistaProcedureCaller caller = mockVistaProcedureCaller();
+        final RpcVistaPatientDao dao = new RpcVistaPatientDao(caller, RADIOLOGIST_DUZ);
+        final Patient patient = dao.getPatient(PATIENT_DFN);
+        assertEquals(Collections.<String>emptyList(), patient.getHealthFactors());
```

Comment:
`getHealthFactors()` is not actually a `List<String>` (anymore). This comparison works due to type erasure, but it's a little misleading.